### ISSUE TITLE
Add Buttons "Open in JOSM/iD/…"

### ISSUE THREAD

**Issue by Nakaner** [tyrasd/overpass-turbo#144]
It would be a great improvement for quality assurance work if there were buttons to open _the current map view_ in JOSM/iD/…

Buttons to open the current Overpass API output in JOSM are not necessary because you can already download the data and open it via File -> Open in JOSM.

**Comment by Abbe98** [comment]
Under export it's possible to load the data in JSOM/Level0.

Why would you need to open the current map view? I feel that it isn't within the scope of Overpass Turbo...

**Comment by Nakaner** [comment]
If I do quality assurance work (e.g. searching defect associatedStreet relations), I want to zoom there on the map and open this map view in JOSM to repair them manually.

**Comment by Abbe98** [comment]
The you can load the data using export->JSOM?

**Comment by Nakaner** [comment]
Yes, but I also want to load the whole environment. If I check this relations I want to compare tags of the searched objects and the objects arround (I check if addr:street=* matches between the relation, their buildings and the streets around).

**Comment by Abbe98** [comment]
Try add a query?
